From 3b44fd0d0bbf006f0cfb13f83ad4af2e41f3e82f Mon Sep 17 00:00:00 2001
Message-Id: <3b44fd0d0bbf006f0cfb13f83ad4af2e41f3e82f.1413837856.git.chang-joon.lee@intel.com>
In-Reply-To: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
References: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
From: Adam Rutkowski <adam.j.rutkowski@intel.com>
Date: Thu, 13 Feb 2014 22:00:15 +0100
Subject: [PATCH 061/312] FOR_UPSTREAM [VPG]: drm/i915: Sysfs file for thaw
 notification on Valleyview

Sysfs file for thaw notification is now exposed on Valleyview.

Issue: VIZ-3502
Change-Id: I2f4f7180622e31f108dbdab4271220b7668d8655
Depends-Change-Id: I6f3fbf1ca6ee79647272f9cf260d233b977de11a
Signed-off-by: Adam Rutkowski <adam.j.rutkowski@intel.com>
Reviewed-on: https://android.intel.com/164174
Reviewed-by: Bloomfield, Jon <jon.bloomfield@intel.com>
Reviewed-by: Holmberg, Hans <hans.holmberg@intel.com>
Reviewed-by: Yang, Fei <fei.yang@intel.com>
Reviewed-by: Odeberg, Hans <hans.odeberg@intel.com>
Tested-by: Odeberg, Hans <hans.odeberg@intel.com>
---
 drivers/gpu/drm/i915/i915_sysfs.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/i915/i915_sysfs.c b/drivers/gpu/drm/i915/i915_sysfs.c
index 99799d0..818ecc5 100644
--- a/drivers/gpu/drm/i915/i915_sysfs.c
+++ b/drivers/gpu/drm/i915/i915_sysfs.c
@@ -571,6 +571,7 @@ static const struct attribute *vlv_attrs[] = {
 	&dev_attr_gt_max_freq_mhz.attr,
 	&dev_attr_gt_min_freq_mhz.attr,
 	&dev_attr_vlv_rpe_freq_mhz.attr,
+	&dev_attr_thaw.attr,
 	NULL,
 };
 
-- 
1.7.9.5

